packages/mobile/src/account/__snapshots__/DataSaver.test.tsx.snap

```diff
@@ -238,11 +238,11 @@ exports[`DataSaver renders correctly 1`] = `
                       "lineHeight": 18,
```

Comment:
Snapshot changed as Modal component is being used. Note that redesign has not yet been applied